README.md

```diff
@@ -60,6 +65,10 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+weather = "sunny" assigns the string "sunny" to the declared variable weather.
+weather === "sunny" is a comparator operator that checks whether weather is
+stricly equal to "sunny", retruning false.
```

Comment:
it returns true because the value of weather is "sunny". It is comparing the value of weather, not the name weather